frontend/src/views/LabelView.vue

```diff
@@ -79,4 +84,85 @@ watchEffect(() => {
       />
     </div>
   </v-container>
+  <v-container fluid class="d-flex align-center justify-center my-2">
+    <v-row justify="center">
+      <v-dialog v-model="dialog">
+        <template v-slot:activator="{ props }">
+          <v-btn color="primary" rounded v-bind="props">
+            Frame: {{ imgIndex + 1 }} of {{ framesList.length }}
+          </v-btn>
+        </template>
+        <v-card>
+          <v-card-title>
+            <span class="text-h5"> Select Frame </span>
+          </v-card-title>
+          <v-row class="d-flex align-center justify-center">
```

Comment:
This can be simplified, to `<v-row justify="center">` you will also need to wrap the element in `<v-card-content>`